test/pstl_offload/general/selected_different_device.pass.cpp

```diff
@@ -0,0 +1,43 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#if __SYCL_PSTL_OFFLOAD__ == 1
```

Comment:
Can we enabling skipping on the CMake level? I guess we are trying to do that. Did I misunderstand?

Reply:
It's definitely possible but requires ugly changes in cmake, while skipping on the test level is straightforward, so I'm for the current implementation.